src/zope/proxy/__init__.py

```diff
@@ -314,11 +302,6 @@ def __floordiv__(self, other):
         return self._wrapped // other
 
     def __truediv__(self, other):  # pragma: no cover
```

Comment:
The `no cover` here, at `__rtruediv__` and at `__itruediv__` look strange. As there now is only one method and no longer two for different Python versions, I think it should be covered or at least be coverable.

Reply:
You are right, they are covered now and I removed the pragma.